src/modelfree/train.py

```diff
@@ -292,22 +305,26 @@ def wrappers_config(env_name):
 
 
 @train_ex.capture
-def build_env(out_dir, _seed, env_name, num_env, victim_type, victim_index, debug):
+def build_env(out_dir, _seed, env_name, num_env, victim_type, victim_index,
+              debug, lookback_params):
     pre_wrapper = GymCompeteToOurs if env_name.startswith('multicomp/') else None
+    resettable = lookback_params['num_lb'] > 0
 
     if victim_type == 'none':
         our_idx = 0
     else:
         our_idx = 1 - victim_index
 
     def env_fn(i):
-        return utils.make_env(env_name, _seed, i, out_dir, our_idx, pre_wrapper=pre_wrapper)
+        return utils.make_env(env_name, _seed, i, out_dir, our_idx, pre_wrapper=pre_wrapper,
+                              resettable=resettable)
 
     if not debug and num_env > 1:
         make_vec_env = make_subproc_vec_multi_env
     else:
         make_vec_env = make_dummy_vec_multi_env
     multi_venv = make_vec_env([functools.partial(env_fn, i) for i in range(num_env)])
+    multi_venv = DebugVenv(multi_venv)
```

Comment:
I don't think we should be doing this by default (maybe only if `debug` flag is set), and I'm a little hesitant about having it here at all. Does it have any use in the non-lookback case?

Reply:
Well we could have it there `if debug and lookback_params['num_lb'] > 0`